recon_surf/README.md

```diff
@@ -4,11 +4,10 @@ This directory contains all information needed to run the surface-generation and
 approximately 1-1.5 h (depending on processing parallelization and image quality) this pipeline provides a fast FreeSurfer 
 alternative for cortical surface reconstruction, mapping of cortical labels and traditional point-wise and ROI thickness analysis.
 The basis for the reconstruction pipeline is the accurate anatomical whole brain segmentation following the DKTatlas 
-such as the one provided by the FastSurferCNN deep learning architecture.
+such as the one provided by the FastSurferCNN or FastSurferVINN deep learning architectures.
```

Comment:
This documentation probably needs some details on the behavior w.r.t. hires processing.
Currently, there is no dedicated flag, but all segmentation files that are passed, which have a voxel size smaller than 0.999 are treated as hires.